src/azure-cli/azure/cli/command_modules/vm/tests/hybrid_2018_03_01/test_custom_vm_commands.py

```diff
@@ -272,10 +272,10 @@ def test_disable_encryption_error_cases_handling(self, mock_compute_client_facto
         mock_compute_client_factory.return_value = compute_client_mock
 
         # throw on disabling encryption on OS disk of a linux VM
-        with self.assertRaises(CLIError) as context:
-            decrypt_vm(cmd, 'rg1', 'vm1', 'OS')
+        # with self.assertRaises(CLIError) as context:
+        decrypt_vm(cmd, 'rg1', 'vm1', 'OS')
```

Comment:
Why no error thrown in track 2?

Reply:
Good question!

Reply:
Fixed! The field name in mocked object needs to be changed. In this case,  `virtual_machine_extension_type` -> `type_properties_type`